### PR TITLE
Update glossing languages

### PR DESCRIPTION
**Pull request by Danble**
#### Relevant Issue
Some people are planning to work on a new dictionary and they requested the following glosses:
Tiriyó
Hixkaryana
Waiwai
Wayana
Apalaí

#### How can the changes be tested? 
/create-dictionary
https://living-dictionaries-git-update-glossing-languages-livingtongues.vercel.app/create-dictionary

#### Checklist before marking ready to merge
Please keep it in draft mode until these are completed:
- [ ] Equal time was spent cleaning the code as writing it (Boy Scout Rule)
  - [ ] Functions
    - [ ] Functions that don't belong in Svelte components are extracted out into `.ts` files
    - [ ] Functions are short and well named
    - [ ] Concise tests are written for all functions
  - [ ] Classes (a Svelte Component is a Class)
    - [ ] Svelte components are broken down into smaller components so that each component is responsible for one thing (Single Responsibility Principle)
    - [ ] Stories/variants are written to describe use cases
  - [ ] Comments are only included when absolutely necessary information that cannot be explained in code is needed

### CODE REVIEW

File: packages/site/src/lib/glosses/glossing-languages.ts [hunk 1]
```diff
@@ -943,6 +943,24 @@ export const glossingLanguages: IGlossLanguages = {
   zu: {
     vernacularName: 'isiZulu',
   },
+  tri: {
```
Comment (Danble) [inline packages/site/src/lib/glosses/glossing-languages.ts:946]: I added manually the new glosses, but I'm not sure if that's the right way.

**Comment by Danble** [comment]
@jacob-8 I also noticed that the bcp codes appear in parentheses and no other gloss shows that behavior. 
What's wrong here?
image

File: packages/site/src/lib/glosses/glossing-languages.ts (hunk 1 above)
Comment (jacob-8) [reply packages/site/src/lib/glosses/glossing-languages.ts:946]: The should be added in the i18n spreadsheet. That will be transferred into the JSON from which you copy into here. But now Vite supports JSON natively without needing a plugin so I will adjust things here to just use the JSON directly. So then we will just need to update the i18n spreadsheet as our source of truth.

**Review by jacob-8 — CHANGES_REQUESTED** [review]
> @jacob-8 I also noticed that the bcp codes appear in parentheses and no other gloss shows that behavior. 
What's wrong here?
image
 

That is because you haven't added these languages to the various locales files under src/locales/gl - all the previous languages have been internationalized.

**Review by jacob-8 — APPROVED** [review]
Looks good

**Pull request merged**